README.md

```diff
@@ -1 +1,179 @@
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [setup and mobile first project] 
+
+This is my first project on Microverse 
+I add a heading to my html file 
+and styled it , by giving it a color.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>JS</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features 
+
+- **[Add HTML]**
+- **[Add css]**
+- **[Add Desktop and Mobile screen responsiveness]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO SECTION-->
+
+## 🚀 Live Demo 
+
+- [Live Demo Link](https://dorcas126.github.io/final-capstone-project-1/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started 
+
+To get a local copy up and running, follow these steps.
+just visit my repo folder (https://github.com/Dorcas126/final-capstone-project-1)
+
+### Prerequisites
+
+In order to run this project you need:
+
+any code editor 
+but mostly use Vs code
+
+### Setup 
+
+Clone this repository to your desired folder:
+https://github.com/Dorcas126/final-capstone-project-1.git
+
+### Install
+
+Install this project with:
+
+Github
+Vs code 
+to be able to access the code
+
+### Usage
+
+To run the project, execute the following command:
+
+Linters
+git commands
+
+### Run tests
+
+To run tests, run the following command:
+
+Linters
+
+<!-- AUTHORS -->
+
+## 👥 Authors 
+
+👤 **Uwimbabazi Dorcas**
+
+- GitHub: [@Dorcas126](https://github.com/Dorcas126)
+- Twitter: [@Dorcasuw](https://twitter.com/Dorcasuwi)
+- LinkedIn: [Uwimbabazi Dorcas](https://www.linkedin.com/in/uwimbabazi-dorcas-956a5a226/)
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing 
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- Future Features -->
+
+### Future Features 
+
+- **[Add functionality using JS]**
+- **[Add Boostrap to my html file]**
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support
+
+If you like this project create an issue or comment on 
+my project , i will be glad to hear from you .
+Thank you.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments 
+
+I would like to thank each and everyone for your support and interest on my project.
+Thank you
```

Comment:
- [ ] Kindly make sure you gave your Acknowledgments to the Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07). as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Original%20design%20idea%20by%20Cindy%20Shin%20in%20Behance.) 😃